### DISCUSSION TITLE
Linux Facebook group

### DISCUSSION THREAD

**Discussion by ShaunTheSalvo** [shadichy/systemback-archlinux#8] [General]
Hi Shadichy,

I just wanted to ask if it would be okay with you, if I were to post in a Linux users' Facebook group I'm a member of, to help let more people know about your version of Systemback.

It's a publicly-available group - https://www.facebook.com/groups/linux.fans.group/

Thanks - Shaun

**Comment by shadichy** [comment]
Hi Shaun,

> It's a publicly-available group - https://www.facebook.com/groups/linux.fans.group/

I've already joined that group before. Will write a post about my systemback port (or translate my existing one from another groups :))!

Thank you anyway - Shadichy
